compute_sdk/globus_compute_sdk/sdk/utils/__init__.py

```diff
@@ -85,14 +85,13 @@ def check_version(task_details: dict | None, check_py_micro: bool = True) -> str
         sdk_dill = client_details["dill_version"]
         if python_mismatch or sdk_dill != worker_dill:
             return (
-                "Local SDK and remote worker execution environment differences "
-                f"detected for endpoint {worker_epid} -\n"
+                "\nEnvironment differences detected between local SDK and "
+                f"endpoint {worker_epid} workers:\n"
                 f"  SDK uses Python {sdk_py}/Dill {sdk_dill} "
                 f"but worker used {worker_py}/{worker_dill}\n"
                 f"  (worker SDK version: {worker_sdk}; worker OS: {worker_os})\n"
```

Comment:
We may want to drop this so that we don't indicate that we expect different SDK or OS versions to cause serialization issues (assuming Python and dill are the same).

Reply:
Different OS versions do matter.  I saw this when doing diagnostics - 
 
``` ... running a 3.12.2 macOS serialized function on the Tutorial MEP (3.11.8) works from Windows, but not a 3.12.4 Windows 11 serialized function to the same Tutorial MEP. ```

But adding both OS and SDK versions for the SDK will make this message very cluttered (in addition to the worker ones).  So I took it out as you suggested to lessen confusion.